src/framebus.ts

```diff
@@ -33,11 +34,13 @@ export class Framebus {
   private verifyDomain?: VerifyDomainMethod;
   private isDestroyed: boolean;
   private listeners: Listener[];
+  private limitBroadCastToOrigin: boolean;
 
   constructor(options: FramebusOptions = {}) {
     this.origin = options.origin || "*";
     this.channel = options.channel || "";
     this.verifyDomain = options.verifyDomain;
+    this.limitBroadCastToOrigin = options.limitBroadCastToOrigin || false;
```

Comment:
Why do we want this to be configurable? Seems like we'd always want this to be `true`. Is there a use case where someone would want the noisy errors?

Reply:
IMO we do always want this, but it also does seem to fall into the category of breaking changes. We didn't want to risk breaking folks current implementations, nor felt appropriate to cut a v6 (since one is currently pretty far along on the `v6.x` branch)

Reply:
Not sure how this would be a breaking change. How would someone catch these errors to do anything based on them? If I recall correctly, `window.postMessage` doesn't actually throw an error in this case. The browser is just logging that it wasn't able to send the message.

Reply:
Yeah that's a great point. And ultimately we're just replicating what is really being done under the hood with `postMessage` but doing it so we don't see errors. Excellent call!